sys/shell/shell.c

```diff
@@ -256,6 +256,9 @@ static int readline(char *buf, size_t size)
         else {
             *line_buf_ptr++ = c;
             _putchar(c);
+#ifdef MODULE_NEWLIB
+            fflush(stdout);
+#endif
```

Comment:
sys/shell/shell.c: otherwise the shell doesn't work correctly (no echo)

doc/doxygen/riot.doxyfile: otherwise the RIOT documentation are corrupted with cc3200sdk doygen directives.

drivers/Makefile.include: I added here some directive to build the cc3200sdk unser drivers (correct me if this is the wrong way)